test/passenger_test.rb

```diff
@@ -66,9 +70,75 @@
         expect(trip.passenger.id).must_equal 9
       end
     end
+
+  end
+
+  #TESTS FOR TOTAL_TIME_SPENT
+  describe "total_time_spent" do
```

Comment:
What happens if the passenger has in progress trips? You must account for this in your methods and test for this edge case.